drivers/ethernet/eth_mcux.c

```diff
@@ -82,7 +82,7 @@ struct eth_context {
 	 * bypassing it and writing a more complex driver working
 	 * directly with hardware).
 	 */
-	u8_t frame_buf[1500];
+	u8_t frame_buf[1500 + 14]; /* Max MTU + ethernet header size */
```

Comment:
any concern about FCS?

Also, how about frame_buf[ETH_DW_MTU + 14]

Reply:
looking at the mcux hal/sdk code looks like FCS isn't included, how about we update the comment to mention FCS not being included.